src/layout/Menu/Menu.style.ts

```diff
@@ -67,58 +80,33 @@ const fade = keyframes`
   }
 `;
 
-export const Wrapper = styled.div<{
+export const ItemStyled = styled.button<{
   active?: boolean;
-  $height?: number;
+  hasAction: boolean;
+  indentation: number;
 }>`
-  border-radius: 0.2rem;
-  position: relative;
-  background: ${({ active, theme }) =>
-    active &&
-    (theme.name === 'dark'
-      ? rgba(theme.content.color, 0.095)
-      : rgba(theme.content.color, 0.065))};
-
-  &:hover {
-    background: ${({ theme }) => rgba(theme.content.color, 0.025)};
-  }
-
-  ${(p) =>
-    p.$height &&
-    css`
-      transition: 230ms ease-in-out;
-      height: ${(p.$height + 1) * 2.5 + 0.25}rem;
-    `}
-
-  > svg {
-    width: 1.125rem;
-    height: 1.125rem;
-    margin: 0 0.5rem 0 0;
-    display: inline-block;
-
-    * {
-      fill: ${({ theme }) => rgba(theme.content.color, 0.667)};
-    }
-  }
-`;
-
-export const ItemStyled = styled.button<any>`
-  padding: 0.5rem 1.25rem;
+  padding: 0.5rem;
+  padding-right: ${(p) => (p.hasAction ? '1.5rem' : '0.5rem')};
+  padding-left: ${({ indentation }) => rem(28 + indentation)};
```

Comment:
Rather than adding `indentation` to a base value here, does it make more sense to subtract a base value from `indentation` on line 35?
```tsx
export const Toggle = styled.button<{
  ...
  left: ${({ indentation }) => rem(indentation)};
  ...
```
https://github.com/vimeo/iris/pull/39/files#diff-26570cb7fb37caa3daa3f83b2a73dfe3aee584da64b9355b489d110faf39c685R35

The way `indentation` works right now, it creates an increasingly larger base padding on the entire menu, rather than incrementing each level by that value.

Current behavior:
<img width="290"  src="https://user-images.githubusercontent.com/319302/131530289-39229b39-ff9a-4e22-9dfe-1777b8a5af56.png">

What I would expect:
<img width="276" src="https://user-images.githubusercontent.com/319302/131530327-e1d05c17-00b0-41ad-8024-5d1cc11c958a.png">

Reply:
Implementation idea to consider raising with the designers: When combining a `Menu` with a resizable `Panel`, we could dynamically adjust the padding increment as the width changes size. It would be easy to prototype this to see if it feels natural.

Reply:
Hmm, I'm not sure I'm seeing the same issue? It looks like as we travel down, each new level of buttons has an additional `8px` of padding.

Reply:
Gotcha, updated the logic behind `indentation` to behave the way you want it
<img width="278" alt="Screen Shot 2021-09-10 at 1 26 52 PM" src="https://user-images.githubusercontent.com/15743757/132893755-2b8b596d-0aaa-4858-9597-60cd6b9deef2.png">